### PR TITLE
Makes hollowed out books silent.

### LINKED ISSUE #2342: stealthy book boxes are not stealthy cause
they emmit box sound when you pick them, put into backpack, interact in any way or form. 
If you see librarian give you Woody Got Wood to give to prisoner and it emmits box sound-it got fucking wirecut, alerm.
Why the fuck boxes even emmit sound when you pick it up

### PR DESCRIPTION
**Pull request by matskuman5**
Fixes #2342 

:cl:
tweak: Hollowed out books no longer make noise like bags when you interact with them.
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/storage/storage.dm [hunk 1]
```diff
@@ -51,7 +52,8 @@
 			if(loc != usr || (loc && loc.loc == usr))
 				return
 
-			playsound(loc, "rustle", 50, 1, -5)
+			if(noisy == 1)
```
Comment (AshCorr) [inline code/game/objects/items/weapons/storage/storage.dm]: `if(noisy)` works fine here

File: code/game/objects/items/weapons/storage/storage.dm [hunk 2]
```diff
@@ -70,7 +72,8 @@
 /obj/item/weapon/storage/proc/content_can_dump(atom/dest_object, mob/user)
 	if(Adjacent(user) && dest_object.Adjacent(user))
 		if(dest_object.storage_contents_dump_act(src, user))
-			playsound(loc, "rustle", 50, 1, -5)
+			if(noisy == 1)
```
Comment (AshCorr) [inline code/game/objects/items/weapons/storage/storage.dm]: and here.

**Review by AshCorr — CHANGES_REQUESTED** [review]
Some things you could do to prettify the code.

**Comment by matskuman5** [comment]
@AshCorr

**Review by AshCorr — APPROVED** [review]

**Pull request closed without merging**